ttml.go

```diff
@@ -197,6 +197,39 @@ func (i *TTMLInItems) UnmarshalXML(d *xml.Decoder, start xml.StartElement) (err
 	return nil
 }
 
+// handleBrTokenReader is used only for decoding TTMLInItems, do not use it anywhere else
+type handleBrTokenReader struct {
+	xml.Decoder
+	holdingToken xml.Token
+}
+
+// Token implements the TokenReader interface, when it meets the "br" tag, it will hold the token and return a newline
+// instead. This is to work around the fact that the go xml unmarshaler will ignore the "br" tag if it's within a
+// character data field.
+func (r *handleBrTokenReader) Token() (xml.Token, error) {
+	if r.holdingToken != nil {
+		returnToken := r.holdingToken
+		r.holdingToken = nil
+		return returnToken, nil
+	}
+
+	t, err := r.Decoder.Token()
+	if err != nil {
+		return nil, err
+	}
+
+	if se, ok := t.(xml.StartElement); ok && strings.ToLower(se.Name.Local) == "br" {
+		r.holdingToken = t
+		return xml.CharData("\n"), nil
+	}
+
+	return t, nil
+}
+
+func newHandleBrTokenReader(r io.Reader) xml.TokenReader {
```

Comment:
Could you change it to `func newTTMLXmlDecoder(ts TTMLInSubtitle) *ttmlXmlDecoder` and use `strings.NewReader("<p>" + ts.Items + "</p>")` inside this function instead of below